app/backend/templates/v2/generated_html/host_reference.html

```diff
@@ -323,7 +323,8 @@
                             <tbody>
                               <tr>
                                 <td align="left" style="font-size:0px;padding:10px 25px;word-break:break-word;">
-                                  <div style="font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:1;text-align:left;color:#333333;">Please go and write a reference for them too. It's a nice gesture and helps us build a community together! When you've both written a reference, both references will become visible. Otherwise {{ from_user.name|v2esc }}'s reference will become visible 2 weeks after the end of your interaction, after which you cannot write a reference back.</div>
+                                  <div style="font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:13px;line-height:1;text-align:left;color:#333333;">Please go and write a reference for them too. It's a nice gesture and helps us build a community together! When you've both written a reference, both references will become visible. Otherwise {{ from_user.name|v2esc
```

Comment:
please regenerate templates afterwards

Reply:
Afterwards meaning after merging the PR?

Reply:
Okay the weird formatting should all be fixed now and I regenerated the html and only see my expected changes.